### PR TITLE
update python/django support: run tests for django 2.2

### PR DESCRIPTION
**Pull request by Niouby**
* run tests for Django 2.0 and Python 3.7
* run tests for Django 2.2 with the current released version instead of rc1

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,12 +1,12 @@
 sudo: no
 language: python
 cache: pip
+dist: xenial
 
 matrix:
   include:
     # Need to use xenial in the docs build for a more recent version of sqlite.
```
Comment (claudep) [inline .travis.yml]: Probably the comment can be removed now, and you also forgot some dist: xenial lines below.

Comment (Niouby) [reply .travis.yml]: Thanks, I remove it. ^^

**Comment by benkonrath** [comment]
Thanks for the PR!

**Comment by Niouby** [comment]
thanks :+1:

**Pull request merged**